src/data/requirements/weapons/assaultRifles.js

```diff
@@ -0,0 +1,74 @@
+import {
+	classic, cliffside, digital, dragon, geometric, skulls, solidColours, sprayPaint, stripes, tiger, woodland
+} from "../camo/_index";
+
+const masteryChallenges = {
+	'Gold': 'Get 3 kills without dying 10 times',
+	'Platinum': 'Get 25 longshot kills',
+	'Polyatomic': 'Get 25 headshot kills',
+};
+export default {
+	'M4': {
+		'Scales': sprayPaint.Scales,
```

Comment:
Can we use consistent styling for how we access object properties? 😄 
```suggestion
		'Scales': sprayPaint["Scales"],
```